### ISSUE TITLE
1.10-RC2: Edit History works not  correct

### ISSUE THREAD

**Issue by oneilak** [osTicket/osTicket#2441]
the customer can not log at the edit history

![user](https://cloud.githubusercontent.com/assets/13413719/9381462/d12edea2-4737-11e5-92d6-7925d758d1b9.png)

and the agent has an incorrect display,
it indicates the changed and not the original text

![agent](https://cloud.githubusercontent.com/assets/13413719/9381461/d12b07b4-4737-11e5-9f2a-36cca69fabbc.png)

**Comment by ntozier** [comment]
I seem to recall that clients/users ability to edit ticket history was by accident and is being removed.
@greezybacon can you confirm or deny this please?

**Comment by oneilak** [comment]
at least the history would come in handy if the customer can see, because of the transparency

**Comment by greezybacon** [comment]
We opted to remove all thread actions, including edits, completely on the client interface for now. And the current rule for the edits is that, if a thread entry is edited and no email is sent out as a result, then the previous item is replaced by the edit. Also, in this case of editing a message, we opted to replace the item in the history, because, suppose the client posted some sensitive information by accident, and it was edited by an agent. It would be important that the sensitive information remain deleted—that is not accessible via the history. Otherwise, the edit feature would be a moot point for editing client-submitted information.
